job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -1329,4 +1343,23 @@ public void run() {
         }, timeout);
         return timer;
     }
+
+    /**
+     * Desc: Waits till the job is Completed
+     * @param jobId
+     * @throws ApiException
+     * @throws InterruptedException
+     */
+    private void waitUntilJobCompletes(String jobId) throws ApiException, InterruptedException {
+        long currentTime= System.currentTimeMillis();
+        long endTime = currentTime+5000;
```

Comment:
Why are you doing this and not using a `for loop` as I suggested? Any tests which have a delay set for the prerequisite jobs of over 5 seconds will fail using this method.

Reply:
Replaced the While loop with a for loop